### PR TITLE
Fix typo in migrating from 3.0 to 4.0 page

### PR DESCRIPTION
**Pull request by SNvMK**
## Summary

Fix typo

## Context
No context available

### CODE REVIEW

**Review by ashmaroli — APPROVED** [review]

**Comment by ashmaroli** [comment]
Thank you @SNvMK 
@jekyllbot: merge +doc

File: docs/_docs/upgrading/3-to-4.md [hunk 1]
```diff
@@ -128,7 +128,7 @@ certain features are desired outside of kramdown's core functionality.
 Out of all the extensions listed in the report linked above, gem
 `kramdown-parser-gfm` is automatically installed along with Jekyll 4.0. The
 remaining extensions will have to be manually installed by the user depending on
-desired funtionality, by listing the extension's gem-name in their `Gemfile`.
```
Comment (luismedicina23) [inline docs/_docs/upgrading/3-to-4.md:131]:
```suggestion
desired funtionality, by listing the extension's gem-name in their `Gemfile`.
```

Comment (luismedicina23) [inline docs/_docs/upgrading/3-to-4.md:131]: History.markdown

**Review by luismedicina23 — COMMENTED** [review]
@ashmaroli

**Pull request merged**